components/Card.js

```diff
@@ -31,49 +32,179 @@ export default function Card({
     setToggleDetailsCard(!toggleDetailsCard);
   }
   return (
-    <>
-      <h2>{name}</h2>
-      <button onClick={() => setShowModalDelete(true)}>X</button>
+    <StyledCard>
+      <StyledH2>
+        {previousState === "doing" ? (
+          <DoneTaskStyle>{name}</DoneTaskStyle>
+        ) : (
+          <>{name}</>
+        )}
+      </StyledH2>
+      <StyledDeleteButton onClick={() => setShowModalDelete(true)}>
+        <DeleteTask />
+      </StyledDeleteButton>
       <ModalDeleteTask
         showModalDelete={showModalDelete}
         handleDelete={handleDelete}
         handleClose={() => {
           setShowModalDelete(false);
         }}
       />
-      <div>
+      <StyledNavDiv>
         {previousState && (
-          <button onClick={moveTaskToPreviousState}>{previousState}</button>
+          <StyledNavButtons column={1} onClick={moveTaskToPreviousState}>
+            {previousState}
+          </StyledNavButtons>
         )}
-        <button onClick={handleClick}>
+        <StyledDetailsButton column={2} onClick={handleClick}>
           {toggleDetailsCard ? "show details" : "hide details"}
-        </button>
+        </StyledDetailsButton>
         {nextState && (
-          <button onClick={moveTaskToNextState}>{nextState}</button>
+          <StyledNavButtons column={3} onClick={moveTaskToNextState}>
+            {nextState}
+          </StyledNavButtons>
         )}
-      </div>
+      </StyledNavDiv>
       {toggleDetailsCard ? null : (
         <>
-          {note && <h2>Notes:</h2>}
-          <p>{note}</p>
-          {time && <h4>Estimated time:</h4>}
-          {time && <p>{time} minutes</p>}
-          <div>
-            <button onClick={() => setShowModalEdit(true)}>edit details</button>
-            <ModalEditCard
-              id={id}
-              name={name}
-              note={note}
-              time={time}
-              tasks={tasks}
-              setTasks={setTasks}
-              updateTask={updateTask}
-              handleClose={() => setShowModalEdit(false)}
-              showModalEdit={showModalEdit}
-            />
-          </div>
+          <StyledDetails>
+            <StyledInput>
+              {note && <StyledH>notes:</StyledH>}
+              <StyledP>{note}</StyledP>
```

Comment:
this kind of styled component names is highly confusing.

i would much more care about _"what is the component used for?"_ than _"what is the html tag of this component?"_.
usually you would trust the author of the component to have chosen the correct semantic tags.